playwright.config.ts

```diff
@@ -0,0 +1,32 @@
+import { devices, PlaywrightTestConfig } from '@playwright/test';
+
+// Config file docs: https://playwright.dev/docs/test-configuration
+const config: PlaywrightTestConfig = {
+  webServer: {
+    command: 'pnpm start',
+    port: 3000,
+    reuseExistingServer: !process.env.CI,
+  },
+  testIgnore: '**/util/__tests__/**',
+  retries: process.env.CI ? 2 : 0,
+  reporter: process.env.CI
+    ? 'list'
+    : [['html', { outputFolder: 'playwright/report/' }]],
+  outputDir: 'playwright/test-results/',
+  use: {
+    trace: 'on-first-retry',
+    screenshot: 'only-on-failure',
+    video: 'retain-on-failure',
+    testIdAttribute: 'data-test-id',
+  },
```

Comment:
Global options for all tests. 
A `trace` of test actions should be recorded on the first retry, `screenshot` should be captured only on test failures, `video` should be retained only on test failures, and a `custom test ID` attribute should be used. The default test ID attribute is `data-testid`

Reply:
yeah for now let's remove the `trace` option, since [we also are removing `retries`](https://github.com/upleveled/next-js-portfolio-web/pull/107/files#r1298510616)

Reply:
removed in this commit
[10f606a](https://github.com/upleveled/next-js-portfolio-web/pull/107/commits/10f606a1ad3d180595e6886d7f020fca3cd484fd)